salt/states/pkg.py

```diff
@@ -165,6 +165,10 @@ def _fulfills_version_spec(versions, oper, desired_version,
     otherwise returns False
     '''
     cmp_func = __salt__.get('pkg.version_cmp')
+    # stripping "with_origin" dict wrapper
+    if salt.utils.is_freebsd():
+        if type(versions) is dict and 'version' in versions:
```

Comment:
@terminalmage @rickyninja and I stared at this for a while to see if we should be doing something inside pkgng.py to disable with_origin but neither of us could determine if that was the right intent.  This seems good to me in all cases but I'm interested if you have any feedback.